samples/templates/partials/env.js

```
@@ -0,0 +1,13 @@
+module.exports = function () {
+  let oktaEnv;
+  try {
+    oktaEnv = require('@okta/env');
+  } catch (err) {
+    if (err.code === 'MODULE_NOT_FOUND') {
+      // try local env module
+      oktaEnv = require('./okta-env');
+    }
```

otherwise it should throw the err?